### PR TITLE
VarshaSaroj: Task 4

### PR DESCRIPTION
**Pull request by VarshaSaroj**

### CODE REVIEW

File: VarshaSaroj/task4.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        p{
+            margin-top: 20px;
+            font-size: 24px;
+        }
+    </style>
+</head>
+<body>
+    <nav class="navbar">
+    <div class="nav">
+        <a href="#home">Home</a>
```
Comment (PradyumnaKrishna) [inline VarshaSaroj/task4.html:29]: This button doesn't works, please check.

File: VarshaSaroj/assets/css/navbar.css
Comment (PradyumnaKrishna) [inline VarshaSaroj/assets/css/navbar.css:1]: Remove this file, not required.

File: VarshaSaroj/f1.c
Comment (PradyumnaKrishna) [inline VarshaSaroj/f1.c:1]: Remove this file.

**Review by PradyumnaKrishna — CHANGES_REQUESTED** [review]
Good job @VarshaSaroj! Just some minor changes.

File: VarshaSaroj/f1.c
Comment (VarshaSaroj) [reply VarshaSaroj/f1.c:1]: ok

**Pull request closed without merging**